receiver/apachereceiver/metadata.yaml

```diff
@@ -76,6 +88,60 @@ metrics:
       monotonic: true
       aggregation: cumulative
     attributes: [ server_name ]
+  apache.cpu.time:
+    enabled: true
+    description: Jiffs used by processes of given category.
+    unit: "{jiff}"
+    sum:
+      value_type: double
+      input_type: string
+      monotonic: true
+      aggregation: cumulative
+    attributes: [ server_name, cpu_level, cpu_mode ]
+  apache.cpu_load:
+    enabled: true
+    description: Current load of the CPU.
+    unit: "{load}"
+    sum:
```

Comment:
Should this be a gauge? (Is it meaningful to aggregate instances of this metric across multiple systems, for example?)

Reply:
That's a good question. In the [otel docs](https://opentelemetry.io/docs/reference/specification/metrics/datamodel/#gauge), gauge is defined as "a sampled value at a given time", which suits the load well.
In addition, I tried to find how similar metrics are treated in other receivers. The only result I found was the [flinkmetricsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/flinkmetricsreceiver):

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/d444f478e7478fdc31c46533136a9ba6bd9f433e/receiver/flinkmetricsreceiver/metadata.yaml#L45-L51

As you can see, it's a gauge there and the unit is `%`. It's seems rational to me and I like the idea of keeping consistency between different components, so I'll change the load here to gauge measured in percents.